cmd/config.go

```diff
@@ -242,6 +242,11 @@ func config(ctx *cli.Context) error {
 				format.MaxClientVersion = new
 				clientVer = true
 			}
+		case "enable-acl":
+			// cannot disable
+			if ctx.Bool(flag) {
+				format.EnableACL = true
```

Comment:
We should update the MinClientVersion once enable ACL

Reply:
By default, new volume created by 1.2 should be accessible by 1.2-, then when ACL is enabled, we should update `MinClientVersion` to `1.2`